src/main/java/io/reactivex/Single.java

```diff
@@ -1736,7 +1740,7 @@
      * </dl>
      * @param <T> the common value type
      * @param <R> the result value type
-     * @param sources the array of SingleSource instances
+     * @param sources the array of SingleSource instances, must not be empty
```

Comment:
"An empty array will result in an `onError` signal of `NoSuchElementException`."